staff/cristian-astudillo/precourse/Bingo Game/index.js

```diff
@@ -0,0 +1,196 @@
+    { number: randomNumber, matched: false },
+    { number: randomNumber, matched: false },
+    { number: randomNumber, matched: false },
+    //next line
+    { number: randomNumber, matched: false },
+    { number: randomNumber, matched: false },
+    { number: randomNumber, matched: false },
+    { number: randomNumber, matched: false },
+    { number: randomNumber, matched: false }
+];
+
+ */
+
+//CREAR CARTÓN BINGO
+//NÚMEROS ALEATORIOS PARA CREAR CARTÓN.
+function randoms() {
+    var number1 = Math.ceil(Math.random() * 40);
+    var number2 = Math.ceil(Math.random() * 40);
+    var number3 = Math.ceil(Math.random() * 40);
+
+    { return number1, number2, number3 };
+};
+
+//FACTORY FUNCTIONS PARA CREAR CARTÓN.
+function numerosCarton() {
+    return {
+        numero1: randoms(), match1: 0,
+        numero2: randoms(), match2: 0,
+        numero3: randoms(), match3: 0
+    };
+}
+
+//CARTON CREADO.
+var numero = 5;
+var cartonBingo = [];
+for (let i = 1; i <= numero; i++) {
+    var carton = { ...numerosCarton() };
+    cartonBingo.push(carton);
+}
+
+//CARTÓN BINGO DEFINITIVO.
+let cartonBingo2 = []
+var linea1 = cartonBingo.map(item => item.numero1);
+var linea2 = cartonBingo.map(item => item.numero2);
+var linea3 = cartonBingo.map(item => item.numero3);
+cartonBingo2.push(linea1, linea2, linea3);
+
+
+//NUMEROS BOMBO SIN REPETICIONES.
+let tombola = [];
+for (let randomNumber = 1; randomNumber < 60; randomNumber++) {
+    let random = Math.ceil(Math.random() * 60);
+    if (tombola.indexOf(random) == -1) {
+        tombola.push(random);
+    } else {
+        randomNumber--;
+    };
+};
+
+//console.log(tombola);
+
+//CONTADOR VUELTAS Y SI HAY COINCIDENCIA DE LÍNEA. 
+let vueltas = 0;
+let lineaMatched = false;
+
+//INICIO BINGO.
+function bingo() {
+    alert("Hola participante!! Bienvenido al Bingo Skylab Coders Game, donde tus sueños se pueden hacer realidad!!..." +
+        "Para poder participar en este concurso, necesitamos algunos datos... ");
+    var nombre = prompt("Nos podrías decir tu nombre ?(Si no contestas o cancelas, el juego se termina de manera abrupta).").toUpperCase();
+    if (nombre === "" || nombre === null) {
+        return alert("Adiós!!");
+    }
+
+    alert(`Bienvenido ${nombre}.`);
+    alert("Las reglas del juego son las siguientes : la pantalla mostrará un cartón de 15 números con 3 líneas de 5 números cada uno, " +
+        " y luego, el número correspondiente del bombo; si coincide con algún número del cartón, se tachará con una 'X', y si tacha alguna línea " +
+        "se cantará 'LÍNEA!!', y si se completa el cartón ... BINGO!!!.");
+
+    alert("A continuación, daremos comienzo al Bingo Skylab Coders Game....");
+    alert("... Y empezamos!!!");
+    alert("A continuación, hacemos entrega de su cartón para poder participar:");
+    alert("CARTÓN BINGO = " + JSON.stringify(cartonBingo2));
+    alert("Ahora daremos comienzo al Bingo Skylab Coders!!...");
```

Comment:
I would recommend the use of one <div> as a message display instead of using several 'alert' methods.
Here is an example:

In the HTML code:

<div id="game_info"></div>

In your javascript code:

document.getElementById('game_info').innerHTML = the messages that you want to display.

Reply:
Ok, thanks for your comment, I'll take a look...In that moment, I didn't know how to use D.O.M. and HTML correctly... Anyway, thanks!!